Demo/UI Tests/American Express UI Tests/AmericanExpress_UITests.swift

```diff
@@ -15,19 +15,25 @@ class AmericanExpress_UITests: XCTestCase {
     }
 
     func testValidCard_receivesRewardsBalance() {
+        waitForElementToBeHittable(app.buttons["Valid card"])
```

Comment:
The Amex test need to be run with a client token (flow does not work with tokenization key). The tests were timing out due to not waiting long enough for the button to appear. Using `waitForElementToBeHittable` allows us to wait for the client token to be generated from the server and the buttons to appear. A sleep has been added to tests where not present for consistency.